trio/testing/_network.py

```diff
@@ -1,7 +1,5 @@
 from .. import socket as tsocket
-from .._highlevel_socket import SocketListener, SocketStream
-
-__all__ = ["open_stream_to_socket_listener"]
+from .._highlevel_socket import SocketStream
```

Comment:
mystery, looks like it's not used. Tests passed. `yapf` won't show anything before and after. But making an error in that code, by removing used `SocketStream`, breaks tests. So I think this is legit change? But want to make sure

Reply:
Yeah, it looks like `SocketListener` wasn't actually used here, so this is fine.